bundles/org.openhab.binding.jablotron/src/main/resources/OH-INF/thing/oasis.xml

```diff
@@ -22,6 +22,7 @@
 			<channel id="lastEventClass" typeId="lastEventClass"/>
 			<channel id="lastEventInvoker" typeId="lastEventInvoker"/>
 			<channel id="lastEventTime" typeId="lastEventTime"/>
+			<channel id="lastEventSection" typeId="lastEventSection"/>
```

Comment:
This would need upgrade instructions:
https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Reply:
I have added an upgrade instruction.

Reply:
Allmost there, the thingTypeVersion needs to be set to the thing properties.

Reply:
OK, I have added the thingTypeVersion property to the oasis thing. thank you.